packages/@aws-cdk/aws-appmesh/test/test.virtual-router.ts

```diff
@@ -123,6 +123,10 @@ export = {
           match: {
             prefixPath: '/',
           },
+          timeout: {
+            idle: cdk.Duration.seconds(8),
+            perRequest: cdk.Duration.seconds(8),
+          },
```

Comment:
No need for this (not used in an assertion).

Reply:
youre right wont bother asserting since its handled in the other test suite, removing